### PR TITLE
Add EE exports

### PR DESCRIPTION
**Pull request by joshri**
Related: https://github.com/weaveworks/weave-gitops-enterprise/issues/3215

This PR handles exports for every OSS import that used a path other than `@weaveworks/weave-gitops`. EE PR will pull in this version of OSS once merged

### CODE REVIEW

File: ui/index.ts [hunk 1]
```diff
@@ -91,7 +93,27 @@ import useNavigation from "./hooks/navigation";
...
+  ListPoliciesRequest,
+  ListPoliciesResponse,
+  GetSessionLogsResponse,
+  ListError,
+  LogEntry,
+  PolicyValidation,
+  PolicyObj,
+  PolicyParam,
+  Core as coreClient,
+} from "./lib/api/core/core.pb";
```
Comment (foot) [inline ui/index.ts:116]: All types are exported in the weave-gitops npm package underneath `/ui/` (would be neat to remove this prefix somehow / someday),  e.g. you can import these from a consuming app with:

```ts
import { GetObjectRequest } from '@weaveworks/weave-gitops/ui/lib/api/core/core.pb';
```

Note this is only for types, not for classes/object/functions etc.

File: ui/index.ts [hunk 2]
```diff
@@ -100,6 +122,7 @@ import {
   Automation,
   Bucket,
   FluxObject,
+  FluxObjectNode,
```
Comment (foot) [inline ui/index.ts:125]: This one looks like a class so have to export it here 👍

File: ui/index.ts (hunk 1 above)
Comment (joshri) [reply ui/index.ts:116]: This is for the eslint rule import/no-unresolved 
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-unresolved.md

At the bottom of that doc under "When to not use this", it says: "If you're using a module bundler other than Node or Webpack, you may end up with a lot of false positive reports of missing dependencies." Maybe that's what we have here?

**Review by jpellizzari — APPROVED** [review]

**Comment by joshri** [comment]
@jpellizzari and I discussed turning off the `no-unresolved` rule in both repos so this PR won't be necessary - closing

**Pull request closed without merging**